devops/Makefile

```diff
@@ -420,14 +420,13 @@ deploy-ha-mongo:
 
 build-forms:
```

Comment:
May be not a part of PR, but I noticed now that the build-forms task is not in order with other build-tasks like build-api, build-workers, build-queue-consumers. this build task is just before it's deploy task.

Reply:
Build forms, does not happen every time like the other builds, as we fetch the unchanged code from the formio repo tag.